gridmenuview/src/androidTest/java/com/bcc/gridmenuview/SquareMenuAdapterTest.java

```diff
@@ -0,0 +1,43 @@
+package com.bcc.gridmenuview;
+import androidx.test.platform.app.InstrumentationRegistry;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import static org.junit.Assert.*;
+
+public class SquareMenuAdapterTest {
+
+    private GridMenuAdapter squareMenuAdapter;
+
+    @Before
+    public void setUp(){
+        squareMenuAdapter = new GridMenuAdapter();
+    }
+
+    @Test
+    public void constructorShouldNotReturnNull(){
+        assertNotNull(squareMenuAdapter);
+    }
+
+    @Test
+    public void sizeShouldReturnThreeAfterAddThreeItem(){
+        ArrayList<MenuItem> list = new ArrayList<>();
+        list.add(new MenuItem("satu", InstrumentationRegistry.
+                getInstrumentation().
+                getContext().
+                getResources().
+                getDrawable(R.drawable.ic_launcher_background)));
+        list.add(new MenuItem("dua", InstrumentationRegistry
+                .getInstrumentation()
+                .getContext()
+                .getResources()
+                .getDrawable(R.drawable.ic_launcher_background)));
+        list.add(new MenuItem("tiga", InstrumentationRegistry.getInstrumentation()
+                .getContext()
+                .getResources()
+                .getDrawable(R.drawable.ic_launcher_background)));
+        squareMenuAdapter.setMenuItems(list);
+        assertEquals(squareMenuAdapter.getItemCount(),3);
```

Comment:
small change, the first parameter is the expectation and the second parameter is the actual value. 
assertEquals(expected, actual)
assertEquals(list.size(), squareMenuAdapter.getItemCount())

Reply:
Please also check the other usage of `assertEquals`, seems like all the `assertEquals` have the arguments reversed.